src/controllers/controllermanager.cpp

```diff
@@ -37,6 +37,12 @@ const int kPollIntervalMillis = 5;
 const int kPollIntervalMillis = 1;
 #endif
 
+// Note: A standard Midi device runs at 31.25 kbps,
+// a usual Midi message has 30 bits so we may not expect more than
+// 1000 messages per second
+// PortMidiController uses a buffer of 64 messages which results in
+// Maximum polling interval of 64 ms a theoritical minimum of 1 ms
+
```

Comment:
OK, that's is correct for bursts, but I am in doubt that Mixxx was ever able to catch Midi messages on that rate. My Assumption on my current issue is like that: RMX 2 controller has gone wild and actually hammer the USB 2.0 bus on full speed. This makes Mixxx unusable and is the party stopper, I am trying to Fix.

Before the patch Mixxx tries to catch up as much messages as possible. After this patch we allow bursts of 64 midi messages at full bus speed, but limit the over all rate to 64000 message per second on Windows and 12800 messages per second on Linux, which is already 64 x the original midi speed. 

Do you think these are reasonable values? 
Do we switch over to a more intelligent approach? 

I will add these infos to the comment.

This applies only to midi, since the other protocols are not polling in terms of "isPolling()"